From linux-usb-owner@vger.kernel.org Mon Feb  4 23:58:14 2008
From: Adrian Bunk <bunk@kernel.org>
Date: Mon, 04 Feb 2008 23:57:45 -0800
Subject: USB: make USB_STORAGE_ONETOUCH available with PM
To: greg@kroah.com
Cc: linux-usb@vger.kernel.org, akpm@linux-foundation.org, bunk@kernel.org, gregkh@suse.de, just.for.lkml@googlemail.com, mdharm-usb@one-eyed-alien.net, n.sillik@temple.edu, stern@rowland.harvard.edu
Message-ID: <200802050757.m157vR0j010923@imap1.linux-foundation.org>


From: Adrian Bunk <bunk@kernel.org>

As Torsten Kaiser pointed out, it seems the dependency of
USB_STORAGE_ONETOUCH on !PM should have been removed in commit
7931e1c6f8007d5fef8a0bb2dc71bd97315eeae9.

Signed-off-by: Adrian Bunk <bunk@kernel.org>
Cc: Matthew Dharm <mdharm-usb@one-eyed-alien.net>
Cc: Alan Stern <stern@rowland.harvard.edu>
Cc: Greg Kroah-Hartman <gregkh@suse.de>
Cc: Torsten Kaiser <just.for.lkml@googlemail.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/storage/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/usb/storage/Kconfig
+++ b/drivers/usb/storage/Kconfig
@@ -123,7 +123,7 @@ config USB_STORAGE_ALAUDA
 
 config USB_STORAGE_ONETOUCH
 	bool "Support OneTouch Button on Maxtor Hard Drives (EXPERIMENTAL)"
-	depends on USB_STORAGE && INPUT_EVDEV && EXPERIMENTAL && !PM
+	depends on USB_STORAGE && INPUT_EVDEV && EXPERIMENTAL
 	help
 	  Say Y here to include additional code to support the Maxtor OneTouch
 	  USB hard drive's onetouch button.
